docs/developer/03-fractal.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: Development
-label: Yarn & Gulp etc.
+label: npm & Gulp etc.
 ---
 
-### Yarn & Gulp
+### npm & Gulp
 
 Commands available in the CLI.
 
-- `yarn start` - start local instance of the fractal build
-- `yarn compile_assets` - compiles all local assets, .js and .css
+- `npm run start` - start local instance of the fractal build
```

Comment:
In NPM, you don't have to type `run` for the `start` task: https://docs.npmjs.com/cli/start - provided it's a task within the `scripts` section of the package.json

Reply:
AFAIK, so for at least 5 years, `start`, `stop` and `test` have always been inbuilt scripts.  I think `prestart` etc. were added more recently.

Reply:
`npm run start` still works, of course

Reply:
Another classic _Ben should read the docs™_ moment
 ლ(ಠ_ಠ ლ